google-cloud-tools-plugin/resources/config.properties

```diff
@@ -15,4 +15,7 @@
 #
 
 # Place cloud tools plugin configuration here.
-feature.appengine.flex=true
\ No newline at end of file
+feature.appengine.flex=true
+
+# Should be used in any Notifications explicitly produced by the plugin.
+notifications.plugin.groupdisplayid=Google App Engine plugin
```

Comment:
Looks like all other usages of Notification just use the appropriate message bundle.